src/diff/props.js

```diff
@@ -149,6 +144,14 @@ export function setProperty(dom, name, value, oldValue, isSvg) {
 	}
 }
 
+function updateAttributeNS(dom, ns, name, value) {
```

Comment:
Let's inline this function like before to save ~10 bytes. Function declarations are always somewhat expensive, so we'll only use them when really needed. The only difference between different callsites is the `ns` parameter, so we can just store it in a variable before this section :+1:

Reply:
Changed it back to inlined.